src/main/java/com/chopshop166/chopshoplib/sensors/InvertDigitalInput.java

```diff
@@ -0,0 +1,50 @@
+package com.chopshop166.chopshoplib.sensors;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+/**
+ * uses DigitalInputSource interface to create a invertable input source (cuz !
+ * is boring)
+ */
+public class InvertDigitalInput extends DigitalInput implements DigitalInputSource {
+
+/**
+ * creates inverted boolean to be used for logic later
+ */
+     * 
```

Comment:
There appears to be a stray `*`, I recommend leading it back to the rest of its family.

Reply:
🗸